src/_gettsim/arbeitslosengeld_2/einkommen.py

```diff
@@ -226,28 +234,12 @@ def nettoquote(
 def anrechnungsfreies_einkommen_m_basierend_auf_nettoquote(
     einkommensteuer__einkünfte__aus_nichtselbstständiger_arbeit__bruttolohn_m: float,
     nettoquote: float,
-    arbeitsl_geld_2_params: dict,
+    parameter_anrechnungsfreies_einkommen_ohne_kinder_in_bg: PiecewisePolynomialParameters,
 ) -> float:
-    """Share of income which remains to the individual.
-
-    Parameters
-    ----------
-    einkommensteuer__einkünfte__aus_nichtselbstständiger_arbeit__bruttolohn_m
-        See basic input variable :ref:`einkommensteuer__einkünfte__aus_nichtselbstständiger_arbeit__bruttolohn_m <einkommensteuer__einkünfte__aus_nichtselbstständiger_arbeit__bruttolohn_m>`.
-    nettoquote
-        See :func:`nettoquote`.
-    arbeitsl_geld_2_params
-        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
-
-    Returns
-    -------
-
-    """
+    """Share of income which remains to the individual."""
     out = piecewise_polynomial(
```

Comment:
```suggestion
    return piecewise_polynomial(
```

Let's get rid of these when we touch them.